x-pack/plugins/canvas/public/components/fullscreen/fullscreen.scss

```diff
@@ -1,7 +1,7 @@
 body.canvas-isFullscreen {
   // this is a hack that overwrites Kibana's core chrome
   .kbnGlobalNav.kbnGlobalNav-isOpen + .app-wrapper,
-  .app-wrapper {
+  .header-global-wrapper + .app-wrapper {
```

Comment:
We seem to have settings of this exact specificity in line 24, could be united

Reply:
Great catch, I've consolidated this and removed selectors related to the old K6 navigation.